_schemaV2.graphql

```diff
@@ -11924,7 +11931,11 @@ type MarketPriceInsightsEdge {
   node: MarketPriceInsights
 }
 
-type MarketingCollection {
+type MarketingCollection implements Node {
```

Comment:
Do you know why MarketingCollection wasn't implementing Node before? I see the interface was added to the type way back